app/models/form/sales/questions/owning_organisation_id.rb

```diff
@@ -16,25 +16,22 @@ def answer_options(log = nil, user = nil)
     return answer_opts unless user
     return answer_opts unless log
 
-    unless user.support?
-      if log.owning_organisation_id.present?
-        answer_opts[log.owning_organisation.id] = log.owning_organisation.name
-      end
+    if log.owning_organisation_id.present?
+      org_value = log.owning_organisation.status == :deleted ? "#{log.owning_organisation.name} (deleted)" : log.owning_organisation.name
```

Comment:
I think this can use the new organisation label function as well? (Plus a few other places)

Reply:
Ah, yes, missed it, thanks!